libs/frontend/feature-render-builder/src/lib/dnd/builder-dnd-component.ts

```diff
@@ -229,6 +229,9 @@ export const BuilderDndComponent = defineComponent({
         // TODO -- is this necessary?
         key: renderKey.value,
         ref: dnd.elementRef,
+        // Make builder dnd component not tab-focusable to avoid jumping between
+        // activeElement and component when on Tab press.
+        tabindex: '-1',
```

Comment:
Hi @sampullman 

This attribute was added because an element will be focused on mouse click, and that seemed to interfere the tab navigation feature in the builder. Without this attribute the builder would jump between activeElement and the next component when Tab is pressed.

https://github.com/samatechtw/pubstudio-builder/assets/59676941/6b60fdd9-aa2e-446f-a5fd-827d3ea1bdf1

Reply:
👍